frontend/src/hooks/UserActions/BecomeSpecialist/useBecomeSpecialist.ts

```diff
@@ -7,24 +7,30 @@ import { BecomeSpecialist } from './BecomeSpecialist';
 export const useBecomeSpecialist = (): BecomeSpecialist => {
     const [finished, setFinished] = useState(false);
     const { user, setUser } = useContext(UserContext);
-    const { state, sendRequest } = useHttp<void>();
+    const { state, sendRequest } = useHttp<BecomeSpecialistResponse>();
 
+    const { response } =state;
     useEffect(() => {
-        if (state.response) {
+        if (response) {
             if (user === null) {
                 throw new Error('User is not logged in');
             }
             setUser({ ...user, isSpecialist: true });
             setFinished(true);
+            window.location.href = response.stripeAccountLink;
         }
-    }, [setUser, state.response, user]);
+    }, [setUser, response, user]);
 
     const doBecomeSpecialist = (): void => {
         if (user === null) {
             throw new Error('User is not authenticated');
         }
         const httpOptions: HttpOptions = {
             method: 'POST',
+            body: {
+                returnUrl: 'http://localhost:3000',
```

Comment:
These hardcoded urls should be at least made constants in 'constants.ts' file for example.